cluster/conf/locators.template

```diff
@@ -14,15 +14,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-# Specify the host name on which a Snappy lead will be started. Also
-# specify the startup directory where the logs and configuration files
-# for that lead instance are kept. Apart from these two things, snappy
-# specific properties can also be passed. If the directory and properties
-# are not specified a default directory is created inside the SPARK_HOME directory.
-#
-#     localhost    -dir=/tmp/data/locator (config args)
+# By default, SnappyData launch scripts will start a single locator on localhost
+# and uses <snappyData_dir>/work/localhost-locator-1/ as the directory for logs and 
+# statistics. 
+# Assuming your network is ssh enabled you can add hostnames (one line per host) to start
+# locator on multiple hosts. 
+#
+# Example configurations ....
+# I) Using bind address : By default, SnappyData binds to all interfaces. 
+# In Cloud environments with an internal IP and a different public IP address for 
+# clients, you should configure the bind-address
+#
+# localhost -client-bind-address=<public IP for clients>
+# Use above line to start a locator on localhost with a specific bind-address
+#
+# II) Logging to different directory 
+# Specify the startup directory where the logs and configuration files
+# for that locator instance are managed. 
+#
+#     localhost    -dir=/tmp/data/locator -client-bind-address=<public IP for clients>
 # 
+# III) Configuring the internal IP address for cluster members to find the locator
+# (TODO: below description is not clear)
 # When a machine is not on any network, specify peer-discovery-address too along with hostname.
 # Also, if there are multiple locators in the cluster, specify hostname:port of other locators.
 # 
```

Comment:
This should better go as item 1) because its the first thing users will have to setup to get the cluster running on a multi-node system.

Remove internal: Configuring the IP address for cluster members to find the locator.

By default locator binds to localhost on port 10334 for discovering other members of the cluster. Set peer-discovery-address to an IP address (usually the internal LAN IP) where other members of cluster can talk to locator (configured as their -locators option), and peer-discover-port if you want to change port from the default 10334. The peer-discovery-address can be a wildcard like 0.0.0.0 to listen on all interfaces. If there are multiple locators in the cluster, then specify hostname:port of other locators in the -locators option.